muckrock/agency/api_v2/serializers.py

```diff
@@ -69,3 +69,35 @@ class Meta:
             "parent",
             "appeal_agency",
         )
+        extra_kwargs = {
+            "id": {"help_text": "The unique identifier for this agency."},
+            "name": {"help_text": "The name of the agency."},
+            "slug": {"help_text": "The slug (URL identifier) for the agency."},
+            "status": {
+                "help_text": (
```

Comment:
The documentation automatically includes the valid values, so they do not need to be specified in the help text

![image](https://github.com/user-attachments/assets/9d993dba-b8f6-4430-91ea-0ad25c417f0d)